lib/router.js

```diff
@@ -114,6 +114,45 @@ Router.prototype.lookupByName = function lookupByName(name, req, res) {
     return route.chain.run.bind(route.chain);
 };
 
+/**
+ * Takes an object of route params and query params, and 'renders' a URL.
+ *
+ * @public
+ * @function render
+ * @param    {String} routeName - the route name
+ * @param    {Object} params -    an object of route params
+ * @param    {Object} query -     an object of query params
+ * @returns  {String} URL
```

Comment:
We could add an example here, the new API site would render it to the website.
Like:
```
@example
server.get( { name: 'cities', path: '/countries/:name/states/:state/cities' }, (req, res, next) => ...);
var cities = server.router.render('cities', {  name: 'Australia', state: 'New South Wales' });
// cities:  '/countries/Australia/states/New%20South%20Wales/cities'
```